docs/BackwardCompatibility.md

```diff
@@ -0,0 +1,241 @@
+e.g. Kubernetes, goLang, Helm and many others.
+
+Fybrik versions are of the form "vX.Y.Z" where X is the major version number, Y is the minor version number
+and Z is the patch release number.
+- Major Releases - released less frequently, indicate that **backward-incompatible public API changes**. This release
+  carries no guarantee that it will be backward compatible with preceding major versions.
+- Minor Releases - indicate **backward-compatible public API changes**. This release guarantees backward compatibility
+  and stability. The backward compatibility policy is defined below, but in general, Fybrik supports 2 previous minor 
+releases.
+- Patch Releases - should not change main functionality, mostly for security or bug fixes, and therefore guarantee
+  backward compatibility.
+
+*Note:* Kubernetes uses similar semantic version model, but its versions do not have the "v" prefix. In order to use
+the same release tag and goLang module tag, we have chosen goLang releases modules pattern with the "v" prefix. The same
+version pattern will be used by all others, except CRDa, versioned Fybrik components. CRDs use its proprietary defined 
+by Kubernetes versioning.   
+
+### Backward compatibility policy
+Fybrik will support up to 2 backward minor releases. For example, the version v1.5.x should be able to work with 
+releases v1.4.x and v1.3.x, on another hand it might work with releases v1.2.x, but it is not guaranteed 
+
+## Fybrik components or features that can influence compatibility 
+
+*Note:* the list is not closed, and can be extended
+
+*Note:* some components, listed below, are external integration points, changes in them have impact not only on the backward 
+compatibility, but on external development process. We emphasized these components with the label `integration point`.  
+
+- Fybrik control plane [Kubernetes Custom Resources](#kubernetes-custom-resource-versions-and-upgrade-plan) 
+`FybrikApplication` is a Fybrik entry point and `FybrikModule` is a definition for externally developed modules, 
+therefore these CRDs are integration points. 
+- [Connectors](#connectors): are Open API services that the Fybrik control plane uses to connect to external systems. 
+These connector services are deployed alongside the Fybrik control plane, and as we can see from their name, all of them 
+are external integration points
+  - Data Catalog connector
+  - Policy Manager connector
+  - Credential Management connector
+- [Default Taxonomy](#default-taxonomy). Terms and values defined by taxonomy are part of Connectors APIs and CRDs 
+validations, therefore we can see taxonomy as part of external connectivity and an integration point.  
+- [Default Policies](#default-policies)
+  - [Data Access Policies](#default-data-access-policies)
+  - [IT Configuration Policies](#default-it-config-policies)
+- [Format of Fybrik log files](#format-of-fybrik-log-files). If Fybrik log files are used by external analytic tools, 
```

Comment:
":" instead of "." to be consistent with connectors

Reply:
again, with connectors we have a list, here is not.